freshmaker/events.py

```diff
@@ -452,3 +452,11 @@ def __init__(self, msg_id, dist_git_branch, container_images,
         self.brew_target = brew_target
         self.requester = requester
         self.requester_metadata_json = requester_metadata_json
+
+
+class BotasChangeStateEvent(ErrataBaseEvent):
+    """ Event triggered, when BOTAS pushes advisory to SHIPPED_LIVE state """
+
+    def __init__(self, msg_id, advisory, dry_run=False):
+        super(BotasChangeStateEvent, self).__init__(msg_id, advisory,
```

Comment:
Optional: Since we are using Python 3, we don't need to specify the class and the instance.
```suggestion
        super().__init__(msg_id, advisory,
```

Reply:
Yes, but since we are using that kind of call everywhere I thought it would be better to have it the same way. But you are right it's more convenient to have it with just 'super()'. I will change it.

Reply:
You can keep it the other way for consistency. Whichever way you prefer.